src/Utils/waitTimeBetweenRetriesConfigs.ts

```diff
@@ -0,0 +1,20 @@
+import { RequestTimeoutConfig } from "../Common/RequestTimeoutConfig";
+
+export const waitTimeBetweenRetriesConfigs : RequestTimeoutConfig =  {
```

Comment:
`RequestTimeoutConfig` as name does not seem to make sense there. Should we have a new interface for this?

Reply:
Since the default wait time was `1000` ms. Increasing the wait time could potentially increase the wait time until a widget gets loaded. Should we keep the values to be `1000` by default then adjust every API call accordingly?